packages/open-next/src/utils/regex.ts

```diff
@@ -0,0 +1,10 @@
+export function getCrossPlatformPathRegex(
+  regex: string,
+  opts: { escape: boolean } = { escape: true },
+) {
+  const newExpr = (
+    opts.escape ? regex.replace(/([[\]().*+?^$|])/g, "\\$1") : regex
+  ).replaceAll("/", "(?:\\/|\\\\)");
```

Comment:
nit/personal pref (`\\\\` makes me 😱)
```suggestion
  ).replaceAll("/", String.raw`(?:\/|\\)`);
```